MCprep_addon/materials/default_materials.py

```diff
@@ -39,7 +43,7 @@ def default_material_in_sync_library(default_material, context):
 	return False
 
 
-def sync_default_material(context, material, default_material, engine):
+def sync_default_material(context: Context, material: Material, default_material: str, engine: Engine) -> Union[Material, str, None]:
```

Comment:
`Union[Type, None]` should be replaced with [`Optional[Type]`](https://docs.python.org/3.7/library/typing.html#typing.Optional) as it's more concise. For this case, I would suggest the following (not just for this, but future cases too):
```suggestion
def sync_default_material(context: Context, material: Material, default_material: str, engine: Engine) -> Optional[Union[Material, str]]:
```

`Optional[Union[Material, str]]` states to the programmer "This function can return either a Material or a string, but it can also return a Nonetype